ethsigner/src/main/java/tech/pegasys/ethsigner/http/JsonRpcHttpService.java

```diff
@@ -121,6 +121,23 @@ private Router router() {
   }
 
   private void handleJsonRpc(final RoutingContext context) {
+    vertx.executeBlocking(
+        future -> {
+          process(context);
+          future.complete();
+        },
+        false,
+        (res) -> {
```

Comment:
Does this response mean we don't need to catch the "Unhandled exception" in process(), as it SHOULD flow through to this handler?

Reply:
It should flow through I think. I'll test it and find out

Reply:
Turns out it will catch any handled exceptions from process, so removed the one in process